blenderproc/python/loader/BlendLoader.py

```diff
@@ -23,6 +23,8 @@ def load_blend(path: str, obj_types: Optional[Union[List[str], str]] = None, nam
                      file's folders, see Blender's documentation for bpy.types.ID for more info) names.
     :param data_blocks: The datablock or a list of datablocks which should be loaded from the given .blend file.
                         Available options are: ['armatures', 'cameras', 'curves', 'hairs', 'images', 'lights', 'materials', 'meshes', 'objects', 'textures']
+
+    :param linked: whether to link instead of a bend datablocs from .blend file.
```

Comment:
The parameter needs some additional documentation about the fact that the objects are locked and cannot be moved